ci-operator/step-registry/ipi/aws/pre/workers-rhel7/provision/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
```

Comment:
maybe just symlink to the i-operator/step-registry/ipi/aws/pre/workers-rhel7/OWNERS

Reply:
https://github.com/openshift/release/blob/master/hack/dedup-owners.sh exists maybe that can help?

Reply:
Will do.

Reply:
Updated.